src/sass/components/_section-promotion.scss

```diff
@@ -0,0 +1,148 @@
+.section-promotion {
+  padding-top: 5px;
+  font-size: 1.25rem;
+  font-weight: 600;
+  margin: 5px;
+
+  .hot-deals {
+    border: 1px solid $black-color;
+    margin: 0;
+    padding: 10px 0;
+    background-color: $black-color;
+    color: $white-color;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0 0 0 1rem;
+    list-style: none;
+
+    li {
+      display: inline-block;
+      margin-left: 0.5rem;
+
+      a {
+        display: block;
+        width: 13px;
+        height: 13px;
+        border-radius: 6px;
+        background-color: #ffffff;
```

Comment:
Kolory do zmiennych. Dotyczy się to całego pliku

Reply:
@lukaszewczak  dodałem wszystkie kolory do zmiennych,